cmd/webhook/main.go

```diff
@@ -0,0 +1,69 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"context"
+	"crypto/tls"
+	"flag"
+	"github.com/golang/glog"
+	"k8s.io/klog"
+	"net/http"
+	"os"
+	"os/signal"
+	"sigs.k8s.io/service-apis/pkg/admission"
+	"syscall"
+)
+
+var (
+	tlsCert, tlsKey string
+)
+
+func main() {
+	flag.StringVar(&tlsCert, "tlsCertFile", "/etc/certs/cert", "File with x509 certificate")
+	flag.StringVar(&tlsKey, "tlsKeyFile", "/etc/certs/key", "File with private key to tlsCertFile")
```

Comment:
Secrets of type `tls` have keys named `tls.key` and `tls.crt`. When these secrets are mounted as files on a container, they map to `<path>/tls.crt` and `<path>/tls.key`. 
I've found defaulting to such a naming scheme useful, because, users then have to ensure that Pod definition is correct and everything else simplify flows through (no need to change CLI flags or anything).

Reply:
ok I can change the default